config/initializers/participatory_spaces_patch.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+Rails.application.config.to_prepare do
```

Comment:
Move this block to the `lib/decidim/superspaces/engine.rb` file. You can see an example here: https://github.com/AjuntamentdeBarcelona/decidim-module-kids/blob/main/lib/decidim/kids/engine.rb

Reply:
As you are using the same code for all the classes, I would create a concern `WithSuperspace` defining this method so I would only need to add `include WithSuperspace` in the classes where I need it.